apps/GroundControl/groundControl.py

```diff
@@ -0,0 +1,13 @@
+#Imports needed
```

Comment:
nit: This is non-blocking feedback (i.e we can go ahead with the merge without making this change), but no need to specify that we will need imports in the future.